Makefile

```diff
@@ -85,9 +91,12 @@ acceptance: ## Run all acceptance tests
 	@cd "$${ACCEPTANCE_WORKDIR}"
 	@go run acceptance/coverage/coverage.go .
 	@$(MAKE) build
+ifneq ($(BUILD_IMG_ARCH),$(BUILD_LOCAL_ARCH))
+	@$(MAKE) build-for-test
+endif
 	@export COVERAGE_FILEPATH="$${ACCEPTANCE_WORKDIR}"
 	@export COVERAGE_FILENAME="-acceptance"
-	@cd acceptance && go test ./...
+	@cd acceptance && go test -timeout $(ACCEPTANCE_TIMEOUT) ./...
```

Comment:
Did it block forever? We should check how long this takes on GitHub, might be longer than 10min...

Reply:
yeah, just over 10. I changed it to 20m. It didn't block forever, but I found it takes much longer on my Mac so I made it configurable.